### PR TITLE
SK-154 Add a CONTRIBUTING file

### PR DESCRIPTION
**Pull request by macieklewkowicz**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+## Submitting pull requests
+
+* For any larger changes or new features create an issue first
+* For small bug fixes you can skip the issue and only create a pull request
+* Make sure to write [clear commit messages](https://cbea.ms/git-commit/) for all your commits
+
+## Support
+
+If you have any questions about this project please start or join a [discussion](https://github.com/KiweeEu/magento2-on-kubernetes/discussions).
```
Comment (tom00) [inline CONTRIBUTING.md:21]: the link should not target MOK I believe.

Comment (macieklewkowicz) [reply CONTRIBUTING.md]: Definitely. Good catch, thanks.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Contributing
```
Comment (tom00) [inline CONTRIBUTING.md:1]: Same as in the MOK, add a section in the main README linking this file.

Comment (macieklewkowicz) [reply CONTRIBUTING.md:1]: See https://github.com/KiweeEu/shopware-storefront-build-config/pull/3/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R20-R24

**Review by tom00 — APPROVED** [review]

**Pull request merged**